crates/rpc/src/handlers/config.rs

```diff
@@ -0,0 +1,31 @@
+use std::sync::Arc;
+
+use alloy_primitives::Address;
+use ream_network_spec::networks::NetworkSpec;
+use serde::{Deserialize, Serialize};
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, with_status},
+};
+
+use super::Data;
+
+#[derive(Serialize, Deserialize, Default)]
+struct DepositContract {
+    #[serde(with = "serde_utils::quoted_u64")]
+    chain_id: u64,
+    address: Address,
+}
```

Comment:
```suggestion
pub struct DepositContract {
    #[serde(with = "serde_utils::quoted_u64")]
    chain_id: u64,
    address: Address,
}

impl DepositContract {
   pub fn new(chain_id: u64, address: Address) -> Self {
       Self {
            chain_id,
            address,
       }
   }
}
```

you might as well make this public and new a new() function